orderfulfiller/order_fulfillment_handler/order_fulfillment_handler.go

```diff
@@ -373,7 +375,18 @@ func (r *orderFulfillmentHandler) InitiateTimeout(ctx context.Context, order db.
 		TxType:   dbtypes.TxTypeInitiateTimeout,
 		TxStatus: dbtypes.TxStatusPending,
 	}); err != nil {
-		return fmt.Errorf("failed to insert raw tx %w", err)
+		return "", fmt.Errorf("failed to insert raw tx %w", err)
 	}
-	return nil
+	return txHash, nil
+}
+
+func (r *orderFulfillmentHandler) SubmitTimeoutForRelay(ctx context.Context, order db.Order, txHash string) error {
+	// the source chain for the relay is the chain that the timeout was
+	// initiated on, which is the orders destination chain
+	initiateTimeoutChain := order.DestinationChainID
+
+	// note that there is no profitability option being passed to the relayer
+	// here, timeouts transfer no value to the solver so if we pass a config
+	// for a min allowed gas % the timeout will never be relayed
+	return r.relayer.SubmitTxToRelay(ctx, txHash, initiateTimeoutChain, hyperlane.RelayOpts{})
```

Comment:
curious if people have thoughts on the comment here? the issue is the normal profitability checks look at the value that the hyperlane relay will give to the relayer, and decides if the gas fee % of the total value is < their specified max fee %. however for timeout relays there is no value being transferred to the solver (the solver just does this at a loss). the only metric I could think of would be to have a minimum gas fee in uusdc or gwei specific to the relay destination chain and just not relay until gas is lower than that amount (or eth price drops if specified in uusdc). but again, relaying the timeouts is kind of 'for the good of the protocol', so maybe we don't want to allow people to wait to submit timeout relays since that would be a worse experience for users?